vnext/template/cpp-app/proj/MyApp.vcxproj

```diff
@@ -14,6 +15,9 @@
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
   </PropertyGroup>
+  <PropertyGroup Label="Version">
```

Comment:
I don't think a version property is necessary (why would a project want to condition behavior based on what original template version was used?).

Reply:
I believe the original ask was it may be useful to have for telemetry (before you finished it), but since that's already done, I removed the version property. If it comes up needed in the future, it's an easy add :)